### DISCUSSION TITLE
When I switch the audio channel to mono from the settings, I hear popping noises in the sound

### DISCUSSION THREAD

**Discussion by SLYILL** [obsproject/obs-studio#9632] [General]
When I switch the audio channel to mono from the settings, I hear popping noises in the sound

**Comment by sebastian-s-beckmann** [comment]
This issue appears to be a support request, for which we don't use GitHub Discussions.
Please make sure to visit the support sections on our [Discord server](https://discord.gg/obsproject) or [Forums](https://obsproject.com/forum).

Thank you!
